lib/srv/desktop/rdp/rdpclient/client_common.go

```diff
@@ -32,16 +32,18 @@ type Config struct {
 	// UserCertGenerator generates user certificates for RDP authentication.
 	GenerateUserCert GenerateUserCertFn
 
+	// Login is the Windows login name.
+	Login string
+	// Width is the initial screen width.
+	Width string
+	// Height is the initial screen Height.
+	Height string
+
 	// AuthorizeFn is called to authorize a user connecting to a Windows desktop.
 	AuthorizeFn func(login string) error
 
-	// TODO(zmb3): replace these callbacks with a tdp.Conn
-
-	// InputMessage is called to receive a message from the client for the RDP
-	// server. This function should block until there is a message.
-	InputMessage func() (tdp.Message, error)
-	// OutputMessage is called to send a message from RDP server to the client.
-	OutputMessage func(tdp.Message) error
+	// TDPConn is the TDP connection.
+	TDPConn *tdp.Conn
```

Comment:
Is `TDPConn` something we can have `connectRDP` create rather than requiring it be provided in the config?